src/tradelocker/tradelocker_api.py

```diff
@@ -1203,7 +1228,9 @@ def create_order(
             else:
                 validity = "IOC"
 
-        if type_ == "limit":
+        if type_ == "limit" or type_=="stop":
```

Comment:
This logic is not ideal (and slightly wrong, right?)

-- it's OK if we gracefully handle the case where e.g. user specifies the stop order but fails to set the stop price, but we should not "train" users to use "price" in all cases.

If should be something along the lines of 

if (type == "stop") && stop_price == None:
    stop_price = price
    self.log.warning(f"Order of {type_ = } specified, but no stop_price set. Using price as stop price")